docs/getting-started/faq.rst

```diff
@@ -48,14 +44,22 @@ Furthermore, some of the requirements are unlikely to be obtainable within xarra
 
 We should note that at least some of our additional requirements, in particular physical units, are being pursued also by the xarray developers.
 
-What about performance?
-~~~~~~~~~~~~~~~~~~~~~~~
+Plotting
+--------
+
+How can I set axis limits when creating a plot?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This can be achieved indirectly with scipp's `generic slicing functionality <../user-guide/slicing.html>`_ and `label-based indexing <../user-guide/slicing.html#Label-based-indexing>`_ in particular.
```

Comment:
I don't know if we need to explain it here, but we should remember that in the case of value based slicing, you may not get an exact match for the boundaries in the case where your binning is quite coarse, because we are finding the closest bin, right?

Reply:
Good point. I don't think we should mention it here, rather think about a solution for nicely selecting the range with binned data. As it is now you would need to define bin edges and the `bin` before plotting.